src/Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsSearchResults.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC, Suspense, useMemo } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { CollectorProfileArtistsSearchResultsQuery } from "__generated__/CollectorProfileArtistsSearchResultsQuery.graphql"
+import { compact } from "lodash"
+import { Message } from "@artsy/palette"
+import {
+  CollectorProfileArtistsListArtist,
+  CollectorProfileArtistsListArtistSkeleton,
+} from "Apps/CollectorProfile/Components/CollectorProfileArtists/CollectorProfileArtistsListArtist"
+
+interface CollectorProfileArtistsSearchResultsProps {
+  term: string
+}
+
+export const CollectorProfileArtistsSearchResults: FC<CollectorProfileArtistsSearchResultsProps> = ({
+  term,
+}) => {
+  return (
+    <Suspense fallback={<CollectorProfileArtistsListPlaceholder />}>
+      <CollectorProfileArtistsSearchResultsArtists term={term} />
+    </Suspense>
+  )
+}
+
+interface CollectorProfileArtistsSearchResultsArtistsProps
+  extends CollectorProfileArtistsSearchResultsProps {}
+
+const CollectorProfileArtistsSearchResultsArtists: FC<CollectorProfileArtistsSearchResultsArtistsProps> = ({
+  term,
+}) => {
+  const { me } = useLazyLoadQuery<CollectorProfileArtistsSearchResultsQuery>(
```

Comment:
Can we use yr client query hook so that we can avoid triggering suspense boundaries?

Reply:
I tried that first. It lead to an impossible to debug problem where sometimes parts of the response were null.

Reply:
ahh dang, all good 👍